From 85f732bb5d20d5cd04a4d0b298fc7778e3a62f45 Mon Sep 17 00:00:00 2001
From: Drew Crawford <drew@sealedabstract.com>
Date: Thu, 14 Apr 2016 21:26:57 -0500
Subject: [PATCH] Don't run libdispatch tests

---
 utils/build-script-impl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/build-script-impl b/utils/build-script-impl
index 1951cd4..01ab44a 100755
--- a/utils/build-script-impl
+++ b/utils/build-script-impl
@@ -2052,7 +2052,7 @@ for deployment_target in "${HOST_TARGET}" "${CROSS_TOOLS_DEPLOYMENT_TARGETS[@]}"
                 pushd "${LIBDISPATCH_BUILD_DIR}"
                 make
                 cd tests
-                make build-tests
+                # make build-tests
                 popd
                 { set +x; } 2>/dev/null
 
-- 
2.6.4 (Apple Git-63)

