From fec9fce023f96a05b0998dbe410a8227e670b7d2 Mon Sep 17 00:00:00 2001
From: Drew Crawford <drew@sealedabstract.com>
Date: Tue, 1 Mar 2016 05:22:01 -0600
Subject: [PATCH 2/2] Update libpwq to upstream master

---
 libpwq | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libpwq b/libpwq
index 4312f99..e16bcd6 160000
--- a/libpwq
+++ b/libpwq
@@ -1 +1 @@
-Subproject commit 4312f994da24d35fb355ad6701418752ab8fdf3a
+Subproject commit e16bcd6c4b4df3fd85939a981baf94b30968a1ba
-- 
2.6.4 (Apple Git-63)

